api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -134,6 +134,10 @@ message BufferSettings {
   // The authorization request will be dispatched and no 413 HTTP error will be returned by the
   // filter.
   bool allow_partial_message = 2;
+
+  // If true, the body sent to the external authorization service is set with raw bytes, instead of
+  // UTF-8 string.
+  bool pack_as_bytes = 3;
```

Comment:
Is this really a buffering option? Seems more of a request body option.

Reply:
True. Since it is tightly coupled with buffering settings, I thought I can put it in here. Anyway, I introduced a new message instead.